lib/s3_meta_sync/syncer.rb

```diff
@@ -260,16 +301,29 @@ def parse_yaml_content(content)
     end
 
     def download_file(source, path, destination, zip)
-      download = download_content("#{source}/#{path}") # warning: using block form consumes more ram
-      download = Zip.unzip(download) if zip
-      path = "#{destination}/#{path}"
-      FileUtils.mkdir_p(File.dirname(path))
+      download = if @config[:acl] == AWS_PRIVATE_ACCESS
```

Comment:
maybe extract a method ?
```suggestion
      download = if private?
```